docs/concepts/flows.md

```diff
@@ -20,14 +22,14 @@ There are two kinds of tasks in Kestra:
 
 Runnable Tasks handle computational work in the flow. For example, file system operations, API calls, database queries, etc. These tasks can be compute-intensive and are handled by [workers](../architecture/#worker). 
 
-By default, Kestra only includes a few Runnable Tasks. However, many of them are available as [plugins](../../plugins/).
+By default, Kestra only includes a few Runnable Tasks. However, many of them are available as [plugins](../../plugins/) and if you use our default Docker image plenty of them will be already included.
 
 
 ### Flowable Task
 
-[Flowable Tasks](../developer-guide/flowable) only handle flow logic (branching, grouping, parallel processing, etc.) and trigger new tasks. For example, the Switch task takes inputs and decides the next tasks to run. 
+[Flowable Tasks](../developer-guide/flowable) only handle flow logic (branching, grouping, parallel processing, etc.) and new task triggers. For example, the Switch task decides the next task to run based on some inputs. 
```

Comment:
take care, don't use the word trigger for everything else than our trigger, start a new tasks ?

Reply:
I changed to 'start new tasks"